javascript-typing-tutor/script.js

```diff
@@ -0,0 +1,67 @@
+var value = 0;
+var attempts = 0;
+var $text = document.querySelectorAll('span');
+var $score = document.querySelector('p');
+var $retry = document.querySelector('button');
+var $modal = document.querySelector('.modal');
+var $modalText = document.querySelector('.modal-text');
+var $background = document.querySelector('body');
+var $opacity = document.querySelector('.text');
+var $retry2 = document.querySelector('.retry2');
+
+document.addEventListener('keydown', keyDown);
+$retry.addEventListener('click', retryButton);
+$retry2.addEventListener('click', modalRetry);
+
+function keyDown(e) {
+  if (e.key === $text[value].innerText) {
+    $text[value].className = 'green-text';
+    attempts++;
+    value++;
+    $score.textContent = 'Score: ' + value + '/27';
+    if (value === 27) {
+      $modal.className = 'modal';
+      $background.className = 'background-color-off';
+      $opacity.className = 'opacity text';
+      $modalText.textContent = Math.round(value / attempts * 100) + '% with ' + (attempts - value) + ' errors.';
+    }
+  } else {
+    $text[value].className = 'wrong-text under-line';
+    attempts++;
+  }
+}
+
+function modalRetry(e) {
+  value = 0;
+  attempts = 0;
+  $modal.className = 'modal hidden';
+  $background.className = '';
+  $opacity.className = 'text';
```

Comment:
Don't know if you missed my last comment, but you can just do this for both retry buttons so that you only have one event handler for "retry"